scripts/bake-dependencies.sh

```
@@ -0,0 +1,175 @@
+
+# Run cmake configure - this downloads WebKit
+cmake $CMAKE_ARGS
+
+# Run cmake build for clone targets only - this downloads Zig, BoringSSL, etc.
+# These are build targets that download dependencies
+print "Downloading build dependencies (Zig, BoringSSL, etc.)..."
+cmake --build "$BUILD_PATH" --target clone-zig clone-boringssl clone-mimalloc clone-zstd clone-lolhtml clone-cares clone-libdeflate clone-libarchive clone-tinycc clone-zlib clone-lshpack clone-brotli clone-highway clone-hdrhistogram clone-picohttpparser || true
+
+# Also download debug WebKit variant for debug builds
+print "Downloading debug WebKit variant..."
+cmake $CMAKE_ARGS -DCMAKE_BUILD_TYPE=Debug -B "$BUN_REPO_PATH/build/debug" || true
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)